solution/backend/regulations/templates/regulations/base.html

```diff
@@ -33,10 +33,17 @@
             {% endblock %}
         </title>
         {% include "regulations/partials/favicon.html" %}
-        {% block spa_styles %}{% endblock %}
+        {% block custom_styles %}
+            <link rel="stylesheet" href="{% static '/bundles/style.css' %}?{{ DEPLOY_NUMBER }}" />
+        {% endblock %}
+        <link
+            rel="stylesheet"
+            href="https://cdn.jsdelivr.net/npm/@mdi/font@4.x/css/materialdesignicons.min.css"
+        />
         <link rel="stylesheet" href="{% static '/css/main.css' %}?{{ DEPLOY_NUMBER }}" />
     </head>
     <body class="ds-base {% block body_class %}{% endblock %}">
+        {% block outside_vue %}{% endblock %}
```

Comment:
new block in which to place complex data that is to be safely passed in to Vue/JS.  See [here for a reminder](https://adamj.eu/tech/2022/10/06/how-to-safely-pass-data-to-javascript-in-a-django-template/#use-json-script-for-complex-values) as to why we do it this way in certain places.